src/electron/main.ts

```diff
@@ -21,13 +22,16 @@ import installExtension, {
 import log from 'electron-log';
 import { autoUpdater } from 'electron-updater';
 
-import nspv from '../util/nspv';
 import { WindowControl } from '../vars/defines';
 import MenuBuilder from './menu';
 
-// unhandled excetions debug
-// const unhandled = require('electron-unhandled');
-// unhandled();
+// loading BitGo and Wasm Cryptoconditions in a separate process
+const getWorkerPath = () =>
+  process.env.NODE_ENV === 'development'
+    ? path.join(app.getAppPath(), 'worker.js')
+    : path.join(app.getAppPath(), '..', '/app/worker.js');
```

Comment:
Are these not the same thing? Doesn't `getAppPath()` return the location of the `app` directory, at which point you're going up a level and then back down into `app`?